TTBot/DataAccess/Events.cs

```diff
@@ -56,5 +56,13 @@ public async Task<EventsWithCount> GetEventByMessageIdAsync(ulong messageId)
                 return await con.SingleAsync<EventsWithCount>(e => e.MessageId == messageId.ToString());
             }
         }
+
+        public async Task<EventsWithCount> GetEventByShortname(ulong guildId, string shortname)
+        {
+            using (var con = await _conFactory.OpenAsync())
+            {
+                return (await con.SelectAsync<EventsWithCount>(ev => (ev.ShortName.ToLower() == shortname.ToLower() && ev.GuildId == guildId.ToString()))).SingleOrDefault();
```

Comment:
You've made the change to make the it filter on the guild Id but there needs to be a further change to the Event creation command to make sure the name is unique accross all channels